README.md

```diff
@@ -10,6 +10,110 @@ Centos 6
+`:repository`           | String    | URL to git repository                         | `cannot be nil`
+`:revision`             | String    | Branch or Commit Hash of Repo                 | `master`
+`:virtualenv_path`      | String    | Path to python virtualenv                     | `nil` with a calculated default.
+`:config_template`      | String/Nil| Path to app config template                   | `nil` with a calculated default.
+`:config_destination`   | String/Nil| Path to app config location on server         | `nil` with a calculated default.
+`:config_vars`          | Hash      | Variables for the config template             | `Cannot be nil if config_destination is set, can be empty`
+`:requirements_file`    | String/Nil| Application's dependencies file.              | `nil` with a calculated default.
+`:django_migrate`       | Boolean   | Whether or not to run django migrations       | `false`
+`:django_collectstatic` | Boolean   | Whether or not to run django collectstatic    | `false`
+`:interpreter`          | String    | Python command (python, python3, etc)         | `python`
+`:gunicorn_port`        | Int/Nil   | Port to run gunicorn on                       | `nil`
+
+For more information on some of these resources (and the sane defaults
+python-webapp sets in case of certain values being) see the 'Notes' section
+below.
+
+### Example
+
+**SETUP:** Before you can use the python-webapp cookbook add the following
+informtion to the corresponding files:
+
+* Add `depends 'python-webapp'` to your metadata.rb.
+* Add `cookbook 'python-webapp', git:
+  'git://github.com/osuosl-cookbooks/python-webapp.git', branch: 'master'` to your
+  Berksfile
+
+**RECIPE USAGE:** The following code would go in a `some_recipe.rb` file:
+
+```
+    proj_path = '/opt/test_app'
+
+    python-webapp 'test_app' do
+      create_user true
+      owner 'test_app'
+      group 'test_app'
+
+      repository 'https://github.com/osuosl/python-test-apps.git'
+      revision 'develop'
+
+      config_template 'config.yml.erb'
+      config_destination "#{path}/config.yml"
+      config_vars(
+          path: proj_path,
+          engine: 'django.db.backends.sqlite3',
+          dbname: "#{path}/yourdatabasename.db"
+      )
+      interpreter 'python2.7'   # can be left out if your systems default python
+                                # interpreter is your interpreter as well
+
+      django_migrate true
+      django_collectstatic true
+
+      gunicorn_port 8888
+    end
+```
+
+In the above example `test_app` should be the name of your python application.
+
+More examples of the python-webapp's usage can be found in this repo's `test`
+directory at
+https://github.com/osuosl-cookbooks/python-webapp/tree/master/test/cookbooks/python-webapp-test
+
+### Notes
+
+**Sane Defaults:**
+* `path` is calculated as `/opt/#{ project_name }`
+* `virtualenv_path` is calculated to be `/opt/venv_#{ project_name }`
```

Comment:
[Not anymore](https://github.com/osuosl-cookbooks/python-webapp/blob/master/providers/default.rb#L47)